addons/copy-message-link/addon.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Copy link button",
+  "description": "Adds \"Copy Link\" button to comments.",
```

Comment:
Maybe explain what exactly the link it copies is? Also, give yourself credit with the credits maifest key.